docs/BindingsOverview.md

```diff
@@ -168,3 +169,93 @@ e.g.
 ```sql
 UPDATE [Products].[az_func].[Leases_<FunctionId>_<TableId>] SET _az_func_AttemptCount = 0 WHERE ProductId = 123
 ```
+
+#### Lease Tables clean up
+
+Before clean up, please see [Leases table](./TriggerBinding.md#az_funcleases_) documentation for understanding how they are created and used.
+
+Why clean up?
+1. You renamed your function/class/method name, which causes a new lease table to be created and the old one to be obsolete.
+2. You created a trigger function that you no longer need and wish to clean up its associated data.
+3. You want to reset your environment.
+The Azure SQL Trigger does not currently handle cleaning up any leftover objects, and so to assist you with that we have provided the below scripts to help guide you through doing that.
+
+- Delete all the lease tables that haven't been accessed in `@CleanupAgeDays` days:
+
+```sql
+-- Deletes all the lease tables that haven't been accessed in @CleanupAgeDays days (set below)
+-- and removes them from the GlobalState table.
+USE <Insert DATABASE name here>;
+DECLARE @TableName NVARCHAR(MAX);
+DECLARE @UserFunctionId char(16);
+DECLARE @UserTableId int;
+DECLARE @CleanupAgeDays int = <Insert desired cleanup age in days here>;
+DECLARE LeaseTable_Cursor CURSOR FOR
+
+SELECT 'az_func.Leases_'+UserFunctionId+'_'+convert(varchar(100),UserTableID) as TABLE_NAME, UserFunctionID, UserTableID
+FROM az_func.GlobalState
+WHERE DATEDIFF(day, LastAccessTime, GETDATE()) > @CleanupAgeDays
+
+OPEN LeaseTable_Cursor;
+
+FETCH NEXT FROM LeaseTable_Cursor INTO @TableName, @UserFunctionId, @UserTableId;
+
+WHILE @@FETCH_STATUS = 0
+BEGIN
+    EXEC ('DROP TABLE IF EXISTS ' + @TableName);
```

Comment:
Let's add print statements to these, put the appropriate one above each statement that drops/deletes (here and below)

`PRINT 'Dropping table ' + @TableName`
`PRINT 'Removing row from GlobalState for UserFunctionID = ' + @UserFunctionId + ' and UserTableID = ' + @UserTableId'`